packages/wonder-stuff-server/src/create-logger.js

```diff
@@ -3,26 +3,19 @@ import winston from "winston";
 import type {NpmLogLevels} from "winston";
 
 import {Errors} from "./errors.js";
-// TODO(somewhatabstract, FEI-4174): Update eslint-plugin-import when they
-// have fixed:
-// https://github.com/import-js/eslint-plugin-import/issues/2073
-// eslint-disable-next-line import/named
-import {Runtime} from "./types.js";
 import type {LoggingOptions, Logger} from "./types.js";
 
 import {getLoggingTransport} from "./get-logging-transport.js";
-import {getRuntimeMode} from "./get-runtime-mode.js";
 
 /**
  * Create a logger with the given options.
  */
 export const createLogger = ({
+    mode,
     level,
     defaultMetadata,
-    mode,
```

Comment:
I initially removed this option entirely, and then I added it back, hence the move in the order.